assets/js/lib/test-utils/factories/hosts.js

```diff
@@ -37,11 +46,69 @@ export const slesSubscriptionFactory = Factory.define(() => ({
   version: '15.3',
 }));
 
+const saptuneServiceActiveEnum = () =>
+  faker.helpers.arrayElement(['enabled', 'disabled', null]);
+
+const saptuneServiceEnabledEnum = () =>
+  faker.helpers.arrayElement(['active', 'inactive', null]);
+
+const saptuneServiceNameEnum = () =>
+  faker.helpers.arrayElement(['tuned', 'sapconf', 'saptune']);
+
+const saptuneSolutionNameEnum = () =>
+  faker.helpers.arrayElement(['NETWEAVER', 'HANA', 'NETWEAVER+HANA']);
+
+const saptuneServiceFactory = Factory.define(() => ({
+  active: saptuneServiceActiveEnum(),
+  enabled: saptuneServiceEnabledEnum(),
+  name: saptuneServiceNameEnum(),
+}));
+
+const saptuneStagingFactory = Factory.define(() => ({
+  enabled: faker.datatype.boolean(),
+  notes: saptuneNotesIDs,
+  solutions_ids: saptuneNotesIDs,
+}));
+
+const saptuneAppliedNotesFactory = Factory.define(() => ({
+  additionally_enabled: faker.datatype.boolean(),
+  id: faker.number.int(),
+}));
+
+const saptuneAppliedSolutionFactory = Factory.define(() => ({
+  notes: saptuneNotesIDs.map((id) => id),
```

Comment:
Really funny this `map` :laughing: 
You are mapping a list to to create a list with the same values hehe

Reply:
A leftover xD thanks :D